ios/CustomerioInAppMessaging.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import React
+
+@objc(CustomerioInAppMessaging)
+class CustomerioInAppMessaging: RCTEventEmitter {
+    public static var shared: CustomerioInAppMessaging?
+
+    @objc override static func requiresMainQueueSetup() -> Bool {
+        false /// false because our native module's initialization does not require access to UIKit
+    }
+
+    override init() {
+        super.init()
+        CustomerioInAppMessaging.shared = self
+    }
```

Comment:
If we set the singleton in-line and make it immutable, we don't need this code in the class anymore.

Reply:
Not making this change due to [this](89#discussion_r1102649595)